packages/api/tests/endpoints/test-rules.js

```diff
@@ -1198,27 +1204,27 @@ test.serial('PATCH nullifies expected fields for existing rule in all datastores
 });
 
 test.serial('PATCH sets SNS rule to "disabled" and removes source mapping ARN', async (t) => {
-  const snsStub = sinon.stub(awsServices, 'sns')
-    .returns({
-      listSubscriptionsByTopic: () => (
-        Promise.resolve({
-          Subscriptions: [{
-            Endpoint: process.env.messageConsumer,
-            SubscriptionArn: randomString(),
-          }],
-        })
-      ),
-      unsubscribe: () => (
-        Promise.resolve()
-      ),
-    });
+  const snsMock = mockClient(awsServices.sns());
```

Comment:
Oh this is neat. I don't know if I knew about `mockClient`.

Reply:
Yeah! That's the new `aws-sdk-client-mock` library we're using. Works great. The only catch is that it stubs all of the service so it's important to explicitly reject calls you're not expecting. e.g. 
```
snsMock
    .onAnyCommand()
    .rejects()
     .on(serviceYouActuallyCareAbout)....
```
That way we're not sending unexpected commands to the mock and having it return `undefined`.